weekendhomework10/src/task1/Client.java

```diff
@@ -0,0 +1,57 @@
+package task1;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.util.Scanner;
+
+/**
+ * This class present the client which send the path of file to server for
+ * checking they are exists
+ * 
+ * @author Niddal.Salkic
+ *
+ */
+public class Client {
+
+	private static final int PORT = Server.PORT;
+
+	public static void main(String[] args) {
+
+		try {
+
+			@SuppressWarnings("resource")
+			Socket client = new Socket("localhost", PORT);
+			BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(
+					client.getOutputStream()));
+			while (true) {
+				@SuppressWarnings("resource")
+				Scanner input = new Scanner(System.in);
+
+				System.out
+						.println("Please insert the path of checking file...");
```

Comment:
"Please insert the path of checking file..." => "Please, insert a file path to check:"

i bilo bi lijepo kursor preba pozicionirati na sljedecu liniju